release.nix

```diff
@@ -192,6 +192,11 @@ let
                   pkgs.lib.optionalString pkgs.stdenv.isDarwin ''
                     export DYLD_LIBRARY_PATH=${grpc}/lib''${DYLD_LIBRARY_PATH:+:}$DYLD_LIBRARY_PATH
                   '';
+
+                shellHook = ''
```

Comment:
Not critical feedback, particularly because we aren't really doing this elsewhere in overridden derivations here but something like: `shellHook = (oldDerivation.shellHook or "") + '' ... '';` can prevent hidden gotchas. Just mentioning it though because I think this is fine.